ops/travis/system-testing/tests/test_swarm_runs.py

```diff
@@ -165,3 +166,24 @@ def test_all_services_up(docker_client, services_docker_compose, tools_docker_co
         "Expected running, got \n{}\n{}".format(
                 pformat(task),
                 get_failed_tasks_logs(running_service, docker_client))
+
+
+async def test_check_serve_root():
+    req = urllib.request.Request("http://localhost:9081/")
+    try:
+        resp = urllib.request.urlopen(req)
+        print("Reason: ", resp)

```

Comment:
question: why don't you check both the response and payload in the one test?
and also is there no status code to check?
you can check doing:
```python
response = urllib.request.urlopen(req)
assert response.getcode() == 200
```

One other thing is that the swarm takes some time to be up (especially on travis), and we need to wait for the services. The test above actually waits. you may need to do the same before testing.

Reply:
Please do not add print in the tests ...